src/unofficial.md

```diff
@@ -25,6 +25,8 @@ Application domains:
 * [CXX — Safe Interop Between Rust and C++](https://cxx.rs)
 * [Triangle From Scratch](https://rust-tutorials.github.io/triangle-from-scratch/) - draw a triangle using Win32, but no external crates
 * [Writing Interpreters in Rust: a Guide](https://rust-hosted-langs.github.io/book/introduction.html)
+* [Writing an OS in Rust](https://os.phil-opp.com/)
```

Comment:
Unfortunately, I'm only listing mdbook format books here. Please could you remove this line?